src/data/pokemon/evolution.h

```diff
@@ -495,23 +495,23 @@ const struct Evolution gEvolutionTable[NUM_SPECIES][EVOS_PER_MON] =
     [SPECIES_KUBFU]                 = {{EVO_LEVEL, 0, SPECIES_URSHIFU},
                                        {EVO_LEVEL, 0, SPECIES_URSHIFU}},
     [SPECIES_RATTATA_ALOLAN]        = {{EVO_LEVEL_NIGHT, 20, SPECIES_RATICATE_ALOLAN}},
-    [SPECIES_SANDSHREW_ALOLAN]      = {{EVO_ITEM, ITEM_NONE,/*ITEM_ICE_STONE*/ SPECIES_SANDSLASH_ALOLAN}},
-    [SPECIES_VULPIX_ALOLAN]         = {{EVO_ITEM, ITEM_NONE,/*ITEM_ICE_STONE*/ SPECIES_NINETALES_ALOLAN}},
+    [SPECIES_SANDSHREW_ALOLAN]      = {{EVO_ITEM, ITEM_ICE_STONE, SPECIES_SANDSLASH_ALOLAN}},
+    [SPECIES_VULPIX_ALOLAN]         = {{EVO_ITEM, ITEM_ICE_STONE, SPECIES_NINETALES_ALOLAN}},
     [SPECIES_DIGLETT_ALOLAN]        = {{EVO_LEVEL, 26, SPECIES_DUGTRIO_ALOLAN}},
     [SPECIES_MEOWTH_ALOLAN]         = {{EVO_FRIENDSHIP, 28, SPECIES_PERSIAN_ALOLAN}},
     [SPECIES_GEODUDE_ALOLAN]        = {{EVO_LEVEL, 25, SPECIES_GRAVELER_ALOLAN}},
     [SPECIES_GRAVELER_ALOLAN]       = {{EVO_TRADE, 0, SPECIES_GOLEM_ALOLAN}},
     [SPECIES_GRIMER_ALOLAN]         = {{EVO_LEVEL, 38, SPECIES_MUK_ALOLAN}},
     [SPECIES_MEOWTH_GALARIAN]       = {{EVO_LEVEL, 28, SPECIES_PERRSERKER}},
     [SPECIES_PONYTA_GALARIAN]       = {{EVO_LEVEL, 40, SPECIES_RAPIDASH}},
-    [SPECIES_SLOWPOKE_GALARIAN]     = {{EVO_ITEM, ITEM_NONE,/*ITEM_GALARICA_CUFF*/ SPECIES_SLOWBRO_GALARIAN},
-                                       {EVO_ITEM, ITEM_NONE,/*ITEM_GALARICA_WREATH*/ SPECIES_SLOWKING_GALARIAN}},
+    [SPECIES_SLOWPOKE_GALARIAN]     = {{EVO_ITEM, ITEM_GALARICA_CUFF, SPECIES_SLOWBRO_GALARIAN},
+                                       {EVO_ITEM, ITEM_GALARICA_WREATH, SPECIES_SLOWKING_GALARIAN}},
```

Comment:
```suggestion
    [SPECIES_SLOWPOKE_GALARIAN]     = {{EVO_ITEM, ITEM_NONE, SPECIES_SLOWBRO_GALARIAN},
                                       {EVO_ITEM, ITEM_NONE, SPECIES_SLOWKING_GALARIAN}},
```
Breaks if merged with Item Expansion since they're not defined in Item Expansion yet.